Marlin/src/lcd/dogm/u8g_dev_tft_320x240_upscale_from_128x64.cpp

```diff
@@ -214,76 +224,97 @@ static const uint8_t ili9341_init_sequence[] = { // 0x9341 - ILI9341
 #if ENABLED(TOUCH_BUTTONS)
 
   static const uint8_t button0[] = {
-     B01111111,B11111111,B11111111,B11111111,B11111110,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00010000,B00000000,B00000001,
-     B10000000,B00000000,B00010000,B00000000,B00000001,
-     B10000000,B01000000,B00010000,B00000000,B00000001,
-     B10000000,B11100000,B00010000,B00000000,B00000001,
-     B10000001,B11110000,B00010000,B00000000,B00000001,
-     B10000011,B11111000,B00010000,B00000000,B00000001,
-     B10000111,B11111100,B00010000,B11111111,B11100001,
-     B10000000,B11100000,B00010000,B11111111,B11100001,
-     B10000000,B11100000,B00010000,B00000000,B00000001,
-     B10000000,B11100000,B00010000,B00000000,B00000001,
-     B10000000,B11100000,B00010000,B00000000,B00000001,
-     B10000000,B11100000,B00010000,B00000000,B00000001,
-     B10000000,B00000000,B00010000,B00000000,B00000001,
-     B10000000,B00000000,B00010000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B10000000,B00000000,B00000000,B00000000,B00000001,
-     B01111111,B11111111,B11111111,B11111111,B11111110,
+     B01111111,B11111111,B11111111,B11111110,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10001111,B11000011,B11000001,B11100001,
+     B10001000,B00000100,B00100010,B00010001,
+     B10001000,B00000100,B00000010,B00000001,
+     B10001000,B00000100,B00000010,B00000001,
+     B10001111,B11000011,B11000010,B00000001,
+     B10001000,B00000000,B00100010,B00000001,
+     B10001000,B00000000,B00100010,B00000001,
+     B10001000,B00000100,B00100010,B00010001,
+     B10001111,B11000011,B11000001,B11100001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B10000000,B00000000,B00000000,B00000001,
+     B01111111,B11111111,B11111111,B11111110,
   };
```

Comment:
what about a cross for ESC?

Reply:
What if "BACK" instead of "X" ?

Reply:
ESC and BACK are english, we cant translate that

Reply:
OK.. we use "X"